CHANGELOG.md

```diff
@@ -39,6 +39,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Fixed
 
+- `opentelemetry-instrumentation-logging` Ensure that logging is properly configured independently of the order of instrumentation. This is especially useful in case of automatic instrumentation)
```

Comment:
I would drop ". This is especially useful in case of automatic instrumentation)"